### PR TITLE
Integration test for find transaction by Id

### PR DESCRIPTION
**Pull request by zetxdev**

### CODE REVIEW

**Review by dawciobiel — APPROVED** [review]
Looks good to me

**Review by Krisu93 — APPROVED** [review]
Looks good to me

File: src/test/java/pl/byczazagroda/trackexpensesappbackend/integration/FindTransactionByIDTest.java [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+                .andExpect(MockMvcResultMatchers.jsonPath("$.statusCode").value(ErrorCode.FT001.getBusinessStatusCode()));
+
+        Assertions.assertEquals(1, financialTransactionRepository.count());
+    }
+
+    private FinancialTransaction createTestFinancialTransaction(Wallet wallet, String description) {
+        return financialTransactionRepository.save(FinancialTransaction.builder()
+                .wallet(wallet)
+                .amount(new BigDecimal("5.0"))
+                .date(Instant.ofEpochMilli(0L))
```
Comment (mirekgab) [inline src/test/java/pl/byczazagroda/trackexpensesappbackend/integration/FindTransactionByIDTest.java:71]:
```suggestion
                .date(Instant.ofEpochMilli(1L))
```
The minimum timestamp value is 1 second.

**Review by odzio33 — APPROVED** [review]
Looks good to me

**Review by dawciobiel — APPROVED** [review]
Looks good to me

**Pull request merged**